solvation_analysis/rdf_parser.py

```diff
@@ -0,0 +1,155 @@
+import numpy as np
+from scipy.interpolate import UnivariateSpline
+import scipy
+import matplotlib.pyplot as plt
+import warnings
+
+
+def interpolate_rdf(bins, rdf, floor=0.05, cutoff=5):
+    """
+    Fits a sciply.interpolate.UnivariateSpline to the starting region of
+    the rdf. The floor and cutoff control the region of the rdf that the
+    spline is fit to.
+
+    Parameters
+    ----------
+    bins : np.array
+        the x-axis bins of the rdf
+    rdf : np.array
+        rdf data matching the bins
+    floor : float
+        the interpolation region begins when the probablility density value exceeds
+        the floor
+    cutoff : float
+        the interpolation region ends when the bins values exceeds the cutoff
+    Returns
+    -------
+    f, bounds : the interpolated spline and the bounds of the interpolation region
+
+    """
+    start = np.argmax(rdf > floor)  # will return first index > rdf
+    end = np.argmax(bins > cutoff)  # will return first index > cutoff
+    bounds = (bins[start], bins[end - 1])
+    f = UnivariateSpline(bins[start:end], rdf[start:end], k=4, s=0)
+    return f, bounds
+
+
+def identify_minima(f):
+    """
+    Identifies the extrema of a interpolated polynomial.
+
+    Parameters
+    ----------
+    f : sciply.interpolate.UnivariateSpline
+
+    Returns
+    -------
+
+    cr_pts, cr_vals : the critical points and critical values of the extrema
+
+    """
+    try:
```

Comment:
I would perhaps change this to:
```python
try:
 isinstance(scipy.interpolate.UnivariateSpline)
except:
	ValueError("blah")
cr_pts = f.derivative.roots() #blah blah
```
This is mainly because I don't like having the majority of work done inside a `try/except` block, but this is a personal thing. If you like it the way it is there really isn't anything wrong with it.